src/Controls/PlanarControls.js

```diff
@@ -521,6 +560,12 @@ class PlanarControls extends THREE.EventDispatcher {
         }
         moveTarget.z = pointUnderCursor.z + targetHeight;
 
+        if (this.camera.isOrthographicCamera) {
+            startZoom = this.camera.zoom;
+            endZoom = startZoom * (1 + Math.min(distanceToPoint / 5000, 1));
```

Comment:
could you explain the 5000?

Reply:
The computation of `endZoom` is inspired from the computation `targetHeight` above, which is the camera height at the end of the travel if camera is perspective. The 5000 is a smoothing distance, which is said to have been determined empirically. I added a comment inspired from the computation of `targetHeight`.